### ISSUE TITLE
ISCE installation: Can we eliminate external libraries

### ISSUE THREAD

**Issue by falkamelung** [geodesymiami/rsmas_insar#79]
Hi Lily,
Our ice installation script `install_isce22.csh` relies on the 4 system libraries  below. Can you  see whether there is any chance to eliminate them and install those libraries with Conda, Pip or anything else without root access?   That would make our system more portable. 
```
MOTIFLIBPATH = /usr/lib64/
X11LIBPATH  = /usr/lib/
MOTIFINCPATH = /usr/include/Xm
X11INCPATH = /usr/include/X11
```

**Comment by falkamelung** [comment]
I saw this comment on the ISCE  GitHub. This means what I want is not possible.

https://github.com/isce-framework/recipes-test/tree/master/recipes/isce2

**Comment by lily-wittle** [comment]
@falkamelung Ah, got it. I'll stop looking into this, then. 

Just to let you know, in my research so far, I had not found anything for `/usr/lib64/` or `/usr/lib`. But I had found that `/usr/include/Xm` needed openmotif or lesstif header files which are usually installed via `sudo apt-get install lesstif2-dev` (or via `yum` on those systems). I had thought that we might be able to use the conda package `openmotif` or `openmotif-dev`. Similarly, `/usr/include/X11` needs X11 header files, usually installed via `sudo apt-get install libx11-dev`, and I had thought we might be able to use the conda package `xorg-libx11`. I did not find a package that specifically had the development headers, and also didn't know if we needed those.
